SER515Project_Team3/src/com/asu/ser515/controller/TeacherServlet.java

```diff
@@ -29,49 +29,50 @@
  */
 
 @SuppressWarnings("serial")
-public class TeacherServlet extends HttpServlet{
-	
+public class TeacherServlet extends HttpServlet {
+
 	public void init(ServletConfig config) throws ServletException {
 		super.init(config);
 	}
-	
+
 	// doPost method to handle form submit coming from web page
-		public void doPost(HttpServletRequest req, HttpServletResponse res) {
-			String quizname=req.getParameter("quizname");
-			String instructions=req.getParameter("instructions");
-			HttpSession session = req.getSession(false);
-			User user = new User();
-			System.out.println("---->"+session.getAttribute("firstname"));
-			user.setFirstName((String) session.getAttribute("firstname"));
-			user.setLastName((String) session.getAttribute("lastname"));
-			user.setUserName((String) session.getAttribute("username"));
-			user.setU_ID((int) session.getAttribute("u_id"));
-			user.setUserType((int) session.getAttribute("usertype"));
-			DBConnServiceImpl serviceImpl = new DBConnServiceImpl();
-			int quizCreated = serviceImpl.quizCreation(user.getU_ID(), quizname, instructions);
-			for( int i = 1; i <= 10; i++) {
-				if(req.getParameter("Question"+i)!=null)  {
-					if(req.getParameter("Solution"+i)!=null) {
-						String question = req.getParameter("Question"+i);
-						String solution = req.getParameter("Solution"+i);
-						QuestionAnswer questionaire = new QuestionAnswer(question, solution);
-						int questionsCreated = serviceImpl.questionaireCreation(user.getU_ID(), questionaire);
-						TeacherServletHelper teacherHelper = new TeacherServletHelper();
-						String teacherRouter = teacherHelper.mapTeacherToPage(quizCreated, questionsCreated);
-						try {
-							req.getRequestDispatcher(teacherRouter).forward(req, res);
-						} catch (IOException ioExc) {
-							ioExc.printStackTrace();
-						} catch (ServletException servletExc) {
-							servletExc.printStackTrace();
-						}
-					}
+	public void doPost(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+		String quizname = req.getParameter("quizname");
+		String instructions = req.getParameter("instructions");
+		HttpSession session = req.getSession(false);
+		User teacher = new Teacher();
+		teacher.setFirstName((String) session.getAttribute("firstname"));
+		teacher.setLastName((String) session.getAttribute("lastname"));
+		teacher.setUserName((String) session.getAttribute("username"));
+		teacher.setUser_Id((int) session.getAttribute("u_id"));
+		teacher.setUserType((int) session.getAttribute("usertype"));
+		DBConnServiceImpl serviceImpl = new DBConnServiceImpl();
+		int quizCreated = serviceImpl.quizCreation(teacher.getUser_Id(), quizname, instructions);
+		
+//		for (int i = 1; i <= 10; i++) {
```

Comment:
Is the code working for storing the quiz question's to DB?
Because as per my understanding this part is where we do that.

Reply:
Changes incorporated